recipes/eigen/bld.bat

```diff
@@ -0,0 +1,6 @@
+mkdir -p build
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release ..
+cmake --build . --target INSTALL --config Release
```

Comment:
Correct it is header only. I copied it from the conda wiki. :laughing: Admittedly, it is probably not appropriate. How should it look?